charts/microgateway/README.md.gotmpl

```diff
@@ -226,431 +231,372 @@ In case that the [Simple DSL configuration](#simple-dsl-configuration) does not
+
+  The example below shows how to install the nginx-ingress-controller with Helm:
+  ```
+  helm repo add stable https://kubernetes-charts.storage.googleapis.com
+  helm install nginx stable/nginx-ingress
+  ```
+
+:information_source: **Note**:<br>
+The Microgateway Helm chart itself does not install the nginx-ingress-controller, but allows to create an Ingress object.
+
+#### Ingress terminating HTTP
+
+  To receive HTTP traffic from the outside of the Kubernetes cluster, use the following configuration:
+  ```
+  ingress:
+    enabled: true
+    annotations:
+      nginx.ingress.kubernetes.io/rewrite-target: /
+      kubernetes.io/ingress.class: nginx
+    hosts:
         - virtinc.com
-```
+  ```
+
+#### Ingress terminating secure HTTPS
+The TLS certificate of the Ingress must be in a secret object which is referred in the Ingress configuration.
+At the time of writing, Ingress supports only the default port 443 for HTTPS and directly assumes it is TLS.
+In case that multiple hosts are configured, TLS-SNI is used to distinguish what host the client requested.
+
+  To receive HTTPS traffic from the outside of the Kubernetes cluster, use the following configuration:
+  ```
+  ingress:
+    enabled: true
+    annotations:
+      nginx.ingress.kubernetes.io/rewrite-target: /
+      kubernetes.io/ingress.class: nginx
+    targetPort: https
+    tls:
+      - secretName: virtinc-tls-secret
+        hosts:
+          - virtinc.com
+  ```
 
 ### Openshift Route
-In Openshift a route is needed to reach the Microgateway from the Internet.
-A route object must be created for its configuration. The creation of such an object is handled by the Microgateway Helm chart.  
-
-#### Without TLS
-This example shows how to configure a route object without TLS using the Helm chart.
-
-```
-route:
-  enabled: true
-  hosts:
-    - virtinc.com
-  targetPort: http
-  tls:
-    enabled: false
-```
-
-#### TLS Configuration
-There are three different TLS configuration options for the openshift route. These are edge, reencrypt and passthrough.  
-Below is a description of how these options can be configured using the Microgateway chart.  
-
-Evtl. nur eine Möglichkeit. 
-There are two different options for specifying TLS certificates in the Helm chart.   
-Either you create a secret (config.tlsSecretName) containing the required certificates or you provide the certificates in a custom-values.yaml file during deployment.   
-
-##### Edge
-With the edge termination TLS is terminated on the router.   
-The TLS certificates are served by the router and must therefore be configured on the router.   
-
-```
-route:
-  enabled: true
-  hosts:
-    - virtinc.com
-  tls:
+The examples are based on the standard Openshift Route. Since this is already available in a Openshift environment, nothing special needs to be done. The Microgateway Helm chart allows to create a Route object.
```

Comment:
Since this is already available in a Openshift environment, othing special needs to be done --> 
Since the Route is already available in a Openshift environment, nothing has to be installed additionally.

Reply:
Done.